pkg/assetvalidator/multivalidator.go

```diff
@@ -0,0 +1,42 @@
+package assetvalidator
+
+import (
+	"context"
+
+	"github.com/asecurityteam/nexpose-asset-attributor/pkg/domain"
+)
+
+// MultiAttributtedAssetValidator is an implementation of AssetValidator which runs specified multiple validations
+// of type AssetValidator. In the event that a company needs different validation checks for an attributed asset, this implementation
+// will handle that
+type MultiAttributtedAssetValidator struct {
```

Comment:
nit: typo here: "Atribbutted" => "Attributed". Also in the comment on line 9.

Reply:
yea there's quite a lot of typos, let me fix them